client/src/App.js

```diff
@@ -22,6 +26,10 @@ function App() {
                     .then(res => res.json())
                     .then(data => setCards(data))
 
+  const search = (searchData) => fetch("http://localhost:4000/search?search=" + searchData)
```

Comment:
Please, be more specific, call this function something like `searchAlbums` instead of `search`, then it's gonna be more clear what it actually does.

And its parameter `searchData`  - better rename it to `text`, coz it's not really some `data` , that's just a plain text. 
Data Model would be an object of keys and values instead.